### DISCUSSION TITLE
Include nested node_modules folder

### DISCUSSION THREAD

**Discussion by thgh** [vercel/vercel#5539] [Help] [answered]
What should the .vercelignore file contain so that I can save files in `src/node_modules`? Currently these files never make it to the builder. (Related to Sapper)

This does not work:

    !src/node_modules

**Comment by aaronblenkush** [comment]
This worked for me for both `.gitignore` and `.vercelignore` for a Sapper project deployed to Vercel.

```
!node_modules/
node_modules/*
!src/node_modules/
```

**Comment by thgh** [comment, marked as answer]
Awesome, could not get your exact ignore file to work, but some trial and error brought me to this:
```
!node_modules
/node_modules/*
```
